0-bootstrap/README.md

```diff
@@ -148,11 +148,12 @@ your current Jenkins manager (controller) environment.
 1. Run `terraform output projects_step_terraform_service_account_email` to get the email address of the admin of step `4-projects`. You need this address in a later procedure.
 1. Run `terraform output cloudbuild_project_id` to get the ID of your Cloud Build project.
 1. Run `terraform output gcs_bucket_tfstate` to get your Google Cloud bucket name from Terraform's state.
-1. Copy the backend:
-   ```
-   cp backend.tf.example backend.tf
+1. Create a `backend.tf` file based on `backend.tf.example` template and update with the name of your Cloud Storage bucket:
```

Comment:
Lets remove https://github.com/terraform-google-modules/terraform-example-foundation/pull/819/commits/9cb659073a4fd4958825836a0f7f4bdb16542576 from this PR as it is unrelated and addressed in https://github.com/terraform-google-modules/terraform-example-foundation/pull/825

Reply:
Done